pkgs/applications/misc/watchmate/default.nix

```diff
@@ -10,22 +10,24 @@
 , wrapGAppsHook4
 , glib
 }:
-
+let appid = "io.gitlab.azymohliad.WatchMate";
```

Comment:
This doesn't seem likely to change, so why not leave it inlined?

Reply:
no real reason other than aesthetics tbh